### PR TITLE
LPS-188906 API Explorer crashes on Firefox

### PR DESCRIPTION
**Pull request by 4lejandrito**
https://liferay.atlassian.net/browse/LPS-188906.

### CODE REVIEW

**Comment by 4lejandrito** [comment]
@izaera 🙄

**Comment by 4lejandrito** [comment]
ci:forward

File: portal-impl/src/com/liferay/portal/servlet/filters/language/LanguageFilter.java [hunk 1]
```diff
@@ -212,7 +212,7 @@ public Enumeration<String> getHeaderNames() {
 					continue;
 				}
 
-				headerNames.add(enumeration.nextElement());
```
Comment (izaera) [inline portal-impl/src/com/liferay/portal/servlet/filters/language/LanguageFilter.java:215]: :scream:

**Comment by izaera** [comment]
thx @4lejandrito :kissing_heart:

**Pull request closed without merging**